dom.bs

```diff
@@ -4118,6 +4264,7 @@ interface Node : EventTarget {
   [CEReactions] Node appendChild(Node node);
   [CEReactions] Node replaceChild(Node node, Node child);
   [CEReactions] Node removeChild(Node child);
+  [CEReactions] Node moveBefore(Node node, Node? child);
```

Comment:
I think we should put this method on `ParentNode`. That would also allow us to simplify the validity checks a bit.

Reply:
It also needs a "web developer" intro box.

Reply:
Done in https://github.com/whatwg/dom/pull/1307/commits/3f3e94c5beda922962dacaeb606087f57bd7f7be! Besides the exposure of the method, the semantics of that commit are not observable, right?